contracts/staking/contracts/src/libs/LibCobbDouglas.sol

```diff
@@ -0,0 +1,93 @@
+/*
+
+  Copyright 2019 ZeroEx Intl.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+*/
+
+pragma solidity ^0.5.9;
+pragma experimental ABIEncoderV2;
+
+import "./LibFixedMath.sol";
+
+
+library LibCobbDouglas {
+
+    /// @dev The cobb-douglas function used to compute fee-based rewards for
+    ///      staking pools in a given epoch. Note that in this function there
+    ///      is no limitation on alpha; we tend to get better rounding on the
+    ///      simplified versions below.
+    /// @param totalRewards collected over an epoch.
+    /// @param ownerFees Fees attributed to the owner of the staking pool.
+    /// @param totalFees collected across all active staking pools in the epoch.
+    /// @param ownerStake Stake attributed to the owner of the staking pool.
+    /// @param totalStake collected across all active staking pools in the
+    ///        epoch.
+    /// @param alphaNumerator Numerator of `alpha` in the cobb-dougles function.
+    /// @param alphaDenominator Denominator of `alpha` in the cobb-douglas
+    ///        function.
+    /// @return ownerRewards Rewards for the owner.
+    function _cobbDouglas(
```

Comment:
No need to prefix with `_` for libraries.

Reply:
I suppose we should change that in `LibFixedMath` too.

Reply:
woops! :+1: